### PR TITLE
Refine OpenVINO Python unit test

### PR DESCRIPTION
**Pull request by qiyuangong**
* Re-enable necessary test.
* Add a new test for `load_openvino` and `predict`.

### CODE REVIEW

File: pyzoo/test/zoo/pipeline/inference/test_inference_model.py [hunk 1]
```diff
@@ -30,13 +30,16 @@
...
+s3_url = "https://s3-ap-southeast-1.amazonaws.com"
 with open(property_path) as f:
     for _ in range(2):  # skip the first two lines
         next(f)
     for line in f:
         if "data-store-url" in line:
             line = line.strip()
             data_url = line.split("=")[1].replace("\\", "")
+if os.getenv('FTP_URI'):
+    s3_url = os.getenv('FTP_URI')
```
Comment (glorysdj) [inline pyzoo/test/zoo/pipeline/inference/test_inference_model.py:42]: we can use data-store-url, i was set it in the pom

Comment (glorysdj) [reply pyzoo/test/zoo/pipeline/inference/test_inference_model.py:42]: please check https://github.com/intel-analytics/analytics-zoo/search?q=data-store-url&unscoped_q=data-store-url

Comment (glorysdj) [reply pyzoo/test/zoo/pipeline/inference/test_inference_model.py:42]: mvn command example
mvn clean test -U \
  -DmembersOnlySuites="com.intel.analytics.zoo.common,com.intel.analytics.zoo.feature,com.intel.analytics.zoo.models,com.intel.analytics.zoo.serving,com.intel.analytics.zoo.pipeline.estimator,com.intel.analytics.zoo.serving,com.intel.analytics.zoo.pipeline.inference,com.intel.analytics.zoo.serving,com.intel.analytics.zoo.pipeline.nnframes,com.intel.analytics.zoo.pipeline.api.autograd,com.intel.analytics.zoo.pipeline.api.keras,com.intel.analytics.zoo.pipeline.api.net" \
  -Dparallel=false \
  -P spark_2.x -Dspark.version=2.1.1 -Dspark-version.project=2.1 \
  -Dbigdl.version=${BIGDL_VERSION} -Ddata-store-url=http://10.239.45.10:8081/repository/raw

**Comment by qiyuangong** [comment]
Jenkins http://10.239.47.210:18888/job/ZOO-PR-Validation/3272/ passed

File: pyzoo/test/zoo/pipeline/inference/test_inference_model.py (hunk 1 above)
Comment (qiyuangong) [reply pyzoo/test/zoo/pipeline/inference/test_inference_model.py]: Yes. We have read data_url from property_path. Have we included `s3` URL in that property?

**Pull request merged**